### PR TITLE
Final Review for Regression Test Post

### PR DESCRIPTION
**Pull request by RRoundTable**
## Before
- Device Dependency
- Device
- Typo: Decktop

## After

- Device Dependency -> Imbalance Resource
- Device -> Machine
- Fix typo: Decktop -> Desktop
image
image
image
image

### CODE REVIEW

**Review by kbc8894 — APPROVED** [review]

File: _posts/2021-02-10-Regresssion-Test.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Pipeline #2의 경우 테스트 요청마다 Docker Image를 빌드해야합니
 반면에, Pipeline #3는 테스트 요청마다 미리 만들어둔 Docker Image를 Pull하여 사용합니다.
 
 
-### Device Dependency
+### Imbalance Resource
```
Comment (yurirocha15) [inline _posts/2021-02-10-Regresssion-Test.md:174]: I believe this should be either "Resource Imbalance" or "Imbalanced Resource"

File: _posts/2021-02-10-Regresssion-Test.md [hunk 2]
```diff
@@ -182,22 +182,22 @@ Pipeline #2의 경우 테스트 요청마다 Docker Image를 빌드해야합니
...
 </figure>
 
-또한 다른 MRX-Decktop2, 3를 보면 컴퓨팅 자원이 여유있다는 것을 알 수 있습니다. 
+또한 다른 MRX-Desktop2, 3를 보면 컴퓨팅 자원이 여유있다는 것을 알 수 있습니다. 
 자원을 효율적으로 사용하기 위해서 남은 자원에 효율적으로 접근하는 것이 필요했습니다. 
 이를 위해서는 Regression Test Pipeline이 특정 자원에 종속되지 않고 필요한 자원에 동적으로 접근하여야 합니다. 
 즉, 위의 [그림8] 예시처럼 Regression Test Pipeline이 특정 자원의 영향을 받는 것을 개선해야합니다.
 
-### Pipeline #4: InDependent on Device
+### Pipeline #4: InDependent on Machine
```
Comment (yurirocha15) [inline _posts/2021-02-10-Regresssion-Test.md:194]: Independent from the Machine

**Review by yurirocha15 — APPROVED** [review]

File: _posts/2021-02-10-Regresssion-Test.md (hunk 1 above)
Comment (RRoundTable) [reply _posts/2021-02-10-Regresssion-Test.md]: Thank you!

File: _posts/2021-02-10-Regresssion-Test.md (hunk 2 above)
Comment (RRoundTable) [reply _posts/2021-02-10-Regresssion-Test.md]: Thank you!

**Pull request merged**